server/generate.py

```diff
@@ -118,8 +118,8 @@ def gen_assignment(course):
     name = course.offering + '/' + display_name.lower().replace(' ', '')
 
     last_night = (datetime.datetime.utcnow()
-                          .replace(hour=0, minute=0, second=0, microsecond=0)
-                          - datetime.timedelta(seconds=1))
```

Comment:
It's whatever but https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator